pypdf/_page_labels.py

```diff
@@ -159,8 +159,7 @@ def index2label(reader: PdfReaderProtocol, index: int) -> str:
             "/A": number2uppercase_letter,
             "/a": number2lowercase_letter,
         }
-        if not isinstance(value, dict):
-            value = reader.get_object(value)
+        # if Nums array not iaw standards and if Nums is empty
```

Comment:
I don't understand that comment. What does "iaw" mean?

Reply:
"In Accordance With" (https://en.wiktionary.org/wiki/IAW) 
I use it often but less than ASAP 😂